### ISSUE TITLE
Request skipping

### ISSUE THREAD

**Issue by cryptix** [dominictarr/epidemic-broadcast-trees#26]
I found a couple of links and mentions about this but I couldn't find a clear explanation altough it sounds simple.. Is it _just_ leaving out unchanged feed in the note/vector clock updates?

**Comment by dominictarr** [comment]
yes. the trick is to record _only what remote definitely said they had_. 
If you request from A5 and I send you A6 and A7, I record that you definitely have A5. (I don't know 100% that you received 7) next time we connect, I check my records, and it says you definitely have A5, I check what I have (A7) it's different, so I mention that.

This time, you have A7 too, cool. I record that. next time, I check, it says A7, that's what I have. so I leave it out. If you have a newer message for A, you'll mention it, then i'll say I have A7, and you'll send it.

Also, if you are not interested in a feed, you say -1 and I record that you don't want that feed. If you change your mind later, you have to mention it.

**Comment by cryptix** [comment]
Thanks. I guess we leave this open until it’s in the readme?

**Comment by dominictarr** [comment]
yes good plan
